.github/workflows/ci.yml

```diff
@@ -155,13 +158,15 @@ jobs:
             Windows*)
               cmake -S . \
                 -B ${{ matrix.build-dir }} \
+                -DCMAKE_INSTALL_PREFIX="$(pwd)/install" \
                 -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} \
```

Comment:
This can be factored out of the platform switch.

Reply:
I can't deal with ` -A x64` right now. Only this (which is needed only for Visual Studio) stops me from having one common 'Configure' for all generators. Adding `-A x64` for all generators leads to CMake error. I'll try to sort out with 'Configure' a bit later

Reply:
I didn't mean a common configure step, only to have a shell variable before the `switch` that stores common cmake options (since you already have an array `args` for non-Windows case, it can be initialized before `switch`).